### PR TITLE
Oracle know rep sta2 july2023

### PR DESCRIPTION
**Pull request by SalenaAshton**

### CODE REVIEW

File: ORACLES/db01_sta.json
Comment (skdebray) [inline ORACLES/db01_sta.json]: JSON doesn't allow comments, so this is not legal JSON.  I think it would be desirable for a file that has a ".json" extension to be legal JSON.  Would it be possible to either remove comments or else move JSON-plus-comments files to a `md` format?

**Comment by SalenaAshton** [comment]
Hi Saumya!

I just removed the comments and committed the changes.

Thanks!
Salena

File: ORACLES/db01_sta.json
Comment (skdebray) [inline ORACLES/db01_sta.json]: These curly braces are unbalanced, so this file is not syntactically correct.  This needs to be fixed.  (**Note:** I didn't just add a bunch of right-curly-braces at the end because I did not know how this would affect which values are associated with which fields.  That needs to be checked.)

REQUEST: Please check any updated JSON files to make sure that they are syntactically correct JSON files.

**Review by skdebray — CHANGES_REQUESTED** [review]

**Comment by SalenaAshton** [comment]
Sorry about that!

Yeah, it looks like there had been one extra curly brace at the end of a
field. Instead of fixing it, I've deleted this whole file so it won't be
pushed onto main. Clay and I talked about removing this file so we can
discuss what I'd like to represent in a list versus a dictionary.

Thanks!
Salena

**Comment by skdebray** [comment]
@SalenaAshton @pawnsac @cl4yton I was about to merge this PR when I realized that it wants to merge `oracle_knowRep_sta2_July2023` into `main`.  But I don't think the Python code for executing and devolving wills has not been updated to work with the new JSON format, and I'd like everything in `main` to be consistent and working as far as possible, so I'd like to have this problem fixed before I merge this PR into `main`.  

I will go ahead and create a new PR to merge `oracle_knowRep_sta2_July2023` into `design-attempt-01`, after which:

- @Muaz: I'll create an issue for fixing the Python code, which I'll assign to you.  
- @SalenaAshton: If the JSON format has now been nailed down, maybe this should be documented somewhere and `README.md` updated to point to it?
- **All of us**: We're at (or very close to) the point where we can create small unit tests for our code.  The wills we've created so far have all been legal.  But maybe we should also think about creating a set of not-legal wills, e.g., where the testator and/or witnesses don't satisfy the legal requirements, so that we can test the validation conditions.

**Pull request closed without merging**
